codecoverage.py

```diff
@@ -91,10 +91,15 @@ def generate_info(grcov_path):
         else:
             ordered_files.append("ccov-artifacts/" + fname)
 
+    # Assume we're on a one-click loaner.
+    mod_env = os.environ.copy()
+    if os.path.isdir('/home/worker/workspace/build/src/gcc/bin'):
```

Comment:
Yeah, I've noticed that too (it actually made me think I was doing something wrong, as I couldn't use gcc 4.9.4 despite adding `/home/worker/workspace/build/src/gcc/bin` to the PATH. Then I noticed it didn't exist, but appeared after a while).
To add some error handling, I need a way to detect if we are in a one-click loaner (because if we aren't and the user is running this script locally, we don't want to fail if that directory doesn't exist). Perhaps I can use the presence of the MH_BRANCH or GECKO_HEAD_REV environment variables as an indication that we're running in a one-click loaner?